mobility_games/games/musical_cane.py

```diff
@@ -9,28 +9,36 @@
 import random
 from visualization_msgs.msg import Marker
 import pyttsx
+import os
 from os import system
 import numpy as np
 import pyaudio
 import wave
 import time
 import sys
 from mobility_games.auditory.audio_controller import *
+from rospkg import RosPack
 
 class AudioFeedback(object):
     def __init__(self):
+        top = RosPack().get_path('mobility_games')
+        self.sound_folder = os.path.join(top, 'auditory/sound_files')
         rospy.init_node('audio_feedback')
         rospy.Subscriber('/tango_pose', PoseStamped, self.process_pose)
+        rospy.Subscriber('/cane_tip', PoseStamped, self.process_cane_tip)
         self.x = None   #   x and y position of Tango
         self.y = None
         self.yaw = None
+        self.cane_x = None
+        self.cane_y = None
+        self.cane_z = None
         self.start = False
         self.player = AudioIO(True)
-        self.tracks = song_sample_1()
-        self.quarter_time = 0.25
+        self.tracks = final_countdown()
+        self.quarter_time = 0.125
```

Comment:
It actually represents the duration of a quarter note, in seconds. Adding comment for clarity.